setup/account_statement_reopen/odoo_addons/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
Just out of curiosity, what do these `setup/` files exactly do? I haven't seen them in any Odoo module so far, so do we really need them?